hangman.py

```diff
@@ -0,0 +1,137 @@
+import random
+import os
+import pickle
+
+### FUNCTIONS ###
+
+
+def display_title():
+    os.system('cls')
+    print("################################")
+    print("#### You're playing hangman ####")
+    print("################################")
+
+
+def get_choice():
+    print("\nEnter [1] to play Hangman. ")
+    print("Enter [2] to add your word to the pool.")
+    print("Enter [3] to see pool of words.")
+    print("Enter [q] to quit game.")
+    choice = input("What would you like to do? ")
+    return choice
+
+
+def load_words():
+    # This function loads names from a file, and puts them in the list 'names'.
+    #  If the file doesn't exist, it creates an empty list.
+    try:
+        file_object = open('words.txt', 'rb')
```

Comment:
It would be good to extract `'words.txt'` into a global variable, so it isn't duplicated in two places (both loading and saving the file).